app/helpers/responses_helper.rb

```diff
@@ -2,4 +2,44 @@ module ResponsesHelper
   def scale_options_for_answers
     ["Strongly Disagree", "Disagree", "Neutral", "Agree", "Strongly Agree"]
   end
+
+  def get_form_field answer_form
+    fieldType = answer_form.object.question.question_type.downcase.gsub(' ', '-')
```

Comment:
Would suggest `field_type` instead of `fieldType` like on the other PR.

Reply:
Happy to merge this otherwise :)

Reply:
Please could you merge that change you made which is now in `develop` into this branch? With `field_type`, then I will merge :)